compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -1125,6 +1125,56 @@
                 }
             ]
         },
+        {
+            "name": "ObjectConstructorExpressionNode",
+            "base": "ExpressionNode",
+            "kind": "OBJECT_CONSTRUCTOR",
+            "attributes": [
+                {
+                    "name": "annotations",
+                    "type": "AnnotationNode",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "objectTypeQualifier",
+                    "type": "Token",
+                    "isOptional": true
+                },
+                {
+                    "name": "objectKeyword",
+                    "type": "Token"
+                },
+                {
+                    "name": "typeDescriptor",
+                    "type": "TypeDescriptorNode",
+                    "isOptional": true
+                },
+                {
+                    "name": "objectConstructorBody",
+                    "type": "ObjectConstructorBodyNode"
+                }
+            ]
+        },
+        {
+            "name": "ObjectConstructorBodyNode",
+            "base": "Node",
+            "kind": "OBJECT_CONSTRUCTOR_BODY",
+            "attributes": [
+                {
+                    "name": "openBraceToken",
+                    "type": "Token"
+                },
+                {
+                    "name": "members",
+                    "type": "Node",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "closeBraceToken",
+                    "type": "Token"
+                }
+            ]
+        },
```

Comment:
I think it is better to merge two nodes. @SupunS WDYT?

Reply:
+1

Reply:
done